src/com/lilithsthrone/game/character/body/valueEnums/TesticleSize.java

```diff
@@ -22,19 +22,31 @@
 	TWO_AVERAGE("average-sized", 2, PresetColour.GENERIC_SIZE_THREE),
 
 	/**Large for a human. Average for a dog or wolf-morph.*/
-	THREE_LARGE("large", 3, PresetColour.GENERIC_SIZE_FOUR),
+	THREE_LARGE("plumb-sized", 3, PresetColour.GENERIC_SIZE_FOUR), //large
```

Comment:
The fruit is spelled 'plum'.

A 'plumb' is a weight hung on the end of a string used to measure vertical-ness.